hyperspy/models/model1d.py

```diff
@@ -710,15 +710,36 @@ def _model2plot(self, axes_manager, out_of_range2nans=True):
             ns[np.where(self.channel_switches)] = s
             s = ns
         return s
-
-    def plot(self, plot_components=False, **kwargs):
+    
+    def _residual2plot(self, axes_manager, out_of_range2nans=True):
```

Comment:
Hyperspy does use the `foo2bar` syntax here and there, but I think its generally considered better to use `foo_to_bar`. So here I'd suggest `_residual_to_plot`, or perhaps better `_residual_for_plot` and `out_of_range_to_nan`.